UnitySDK/Assets/ML-Agents/Scripts/Academy.cs

```diff
@@ -95,7 +95,7 @@ public abstract class Academy : MonoBehaviour
         [SerializeField]
         public BroadcastHub broadcastHub = new BroadcastHub();
 
-        private const string kApiVersion = "API-8";
+        private const string kApiVersion = "API-9";
```

Comment:
What's the procedure on these? We didn't actually change the API for this release, right?

Reply:
We usually just change the API along with the major release (even though they are still compatible).